apps/sms/services/test/unit/mock_bridge.js

```diff
@@ -7,4 +7,6 @@
     client: () => {},
     service: () => {}
   };
+
+  exports.MockstreamService = {};
```

Comment:
I thought we don't need separate file for this shim so placed it here, what do you think?